src/game_renderer/GameRenderer.cpp

```diff
@@ -185,15 +185,244 @@ void GameRenderer::RenderLevelEditor(GLFWwindow* window,
   RenderIt(window, game_state, true);
 }
 
-void GameRenderer::RenderIt(GLFWwindow* window,
+void GameRenderer::RenderMiniMap(GLFWwindow* window,
+                std::shared_ptr<GameState> game_state,
+                bool render_level_editor) {
```

Comment:
I'd like that but I think that's out of scope for this commit https://github.com/bbeck13/RhythmRunner/issues/134